rust/src/treefile.rs

```diff
@@ -703,6 +703,10 @@ impl Treefile {
             .unwrap_or_default()
     }
 
+    pub(crate) fn get_platform_module(&self) -> String {
+        self.parsed.platform_module.clone().unwrap_or_default()
```

Comment:
Should be

```suggestion
        self.parsed.base.platform_module.clone().unwrap_or_default()
```

now.

Reply:
Fixed. Thanks!